tasks/timesyncd.yml

```diff
@@ -4,21 +4,29 @@
   ansible.builtin.package:
     name: systemd-timesyncd
     state: present
+  when: ansible_os_family == "Debian"  
```

Comment:
Why was this added?

Reply:
Why is this change part of commit 573b13ff3951526b98100e4d7c3eccd0379c5907?  The commit message for that commit doesn't mention anything that explains this change.

Reply:
[The commit you referenced](https://github.com/konstruktoid/ansible-role-hardening/commit/e238a9d2be1a0ac8e4b9103e58456e2449d7129f) does not contain this change.  That commit also states that the file _doesn't exist by default_ on Fedora 35; it doesn't say that the package doesn't exist at all.  In any event, [Fedora 35 is EOL as of almost a year ago](https://docs.fedoraproject.org/en-US/releases/eol/).

Also, please see [this comment](https://github.com/cisagov/ansible-role-hardening-2/pull/1#discussion_r1388217368), as it also applies here.

Reply:
I think you should remove this line, in the interest of staying as true to the upstream as possible and therefore making the pulling in of upstream commits in the future as smooth as possible.

Reply:
Removed line in [b353fe0](https://github.com/cisagov/ansible-role-hardening-2/pull/1/commits/b353fe083b358cca07f3cc643adc6ce671121068)